cinch/roles/jenkins_master/defaults/main.yml

```diff
@@ -17,16 +17,6 @@ update_centers:
 # you can override jenkins_ssl_cert and jenkins_ssl_key if you want to use custom
 # SSL certificate, otherwise a self signed certificate will be used.
 https_enabled: false
-# Local path to the SSL certificate to use in configuring HTTPS. This can be
-# overridden if you want secure access to Jenkins using your own SSL certificate.
-# By default, a self signed certificate created by the nginx role will be used.
-#jenkins_ssl_cert:
-# Local path to the SSL encryption key to use in configuring HTTPS. See the
-# comment on jenkins_ssl_cert for more information.
-#jenkins_ssl_key:
-# The version of Jenkins to install from the repository. The only use for
```

Comment:
Because since nginx became an integral part of cinch and it is taking care of the HTTP(S) termination, I left the termination configuration up to the nginx role to remove redundancy, as @robled implied in comments to the previous PR. I believe that makes the code cleaner while not affecting functionality.